src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -35,14 +35,25 @@
     private ResultDisplay resultDisplay;
     private HelpWindow helpWindow;
 
+    private SessionListPanel sessionListPanel;
+
     @FXML
     private StackPane commandBoxPlaceholder;
 
     @FXML
     private MenuItem helpMenuItem;
 
     @FXML
-    private StackPane personListPanelPlaceholder;
```

Comment:
Little unsure of this part, might need @PotatoCombat help on fxml

Reply:
So this variable should match the name given in the fxml file. The @FXML notation helps to link the variable for you.

Reply:
This placeholder is a used to represent an empty but resizable box area in the ui!

StackPane allows you to place things on top of each other.